infrastructure/main.tf

```diff
@@ -123,9 +118,9 @@ module "track-your-appeal-notifications" {
     IDAM_URL = "${data.azurerm_key_vault_secret.idam-api.value}"
     IDAM_API_JWK_URL = "${data.azurerm_key_vault_secret.idam-api.value}/jwks"
 
-    IDAM.S2S-AUTH.TOTP_SECRET  = "${data.azurerm_key_vault_secret.sscs-s2s-secret.value}"
-    IDAM.S2S-AUTH              = "${local.s2sCnpUrl}"
-    IDAM.S2S-AUTH.MICROSERVICE = "${var.idam_s2s_auth_microservice}"
+    IDAM_S2S_AUTH_TOTP_SECRET  = "${data.azurerm_key_vault_secret.sscs-s2s-secret.value}"
+    IDAM_S2S_AUTH              = "${local.s2sCnpUrl}"
+    IDAM_S2S_AUTH_MICROSERVICE = "${var.idam_s2s_auth_microservice}"
```

Comment:
I would remove this assuming it's always sscs and just let your default config pick it up

Reply:
Addressed